### ISSUE TITLE
Undefined array key "HTTP_X_FORWARDED_PREFIX" after update to 1.2.0

### ISSUE THREAD

**Issue by SuitDeer** [mosparo/mosparo#240]
Hello everyone,
since the update to 1.2.0 the css Stylesheet is not loading anymore:

(Mosparo is running as a docker container):

```yaml
version: "3.1"
services:
  db:
    # We recommend the mariadb image
    image: mariadb:10.10
    # If you want to use MySQL, uncomment the following line (and comment the one above)
    #image: mysql:8.0.27
    command: '--default-authentication-plugin=mysql_native_password'
    volumes:
      - db_data:/var/lib/mysql
    restart: always
    environment:
      - MYSQL_ROOT_PASSWORD=<SECURE_PASSWORD>
      - MYSQL_DATABASE=mosparo
      - MYSQL_USER=mosparo
      - MYSQL_PASSWORD=<SECURE_PASSWORD>
    expose:
      - 3306
      - 33060
  mosparo_web:
    image: mosparo/mosparo:latest
    ports:
      - 8090:80
    restart: always
    environment:
      - MOSPARO_ENABLE_WEBSERVER=1
      - MOSPARO_ENABLE_CRON=0
    volumes:
      - mosparo_data:/mosparo-data
  mosparo_cron:
    image: mosparo/mosparo:latest
    restart: always
    environment:
      - MOSPARO_ENABLE_WEBSERVER=0
      - MOSPARO_ENABLE_CRON=1
    volumes:
      - mosparo_data:/mosparo-data
volumes:
  db_data:
  mosparo_data:
```

https://mosparo.hostlab.tech/resources/084d6183-8811-4ab3-9668-0abd5515c9be.css

```
Warning: Undefined array key "HTTP_X_FORWARDED_PREFIX" in /mosparo/public/index.php on line 22

Warning: Cannot modify header information - headers already sent by (output started at /mosparo/public/index.php:22) in /mosparo/public/index.php on line 26

Warning: Cannot modify header information - headers already sent by (output started at /mosparo/public/index.php:22) in /mosparo/public/index.php on line 27
```

I have integrated mosparo into my blog: [https://hostlab.tech](https://hostlab.tech/blog/grav-comments-mosparo)

And I have changed the Hostname in mosparo like so:
![grafik](https://github.com/mosparo/mosparo/assets/114341346/2aeec01b-cab4-42ca-bbfd-a383ddc1880a)

I am using the "nginx proxy manager" docker container as a reverse proxy (mosparo and "nginx proxy manager" are running on the same docker host system):

![grafik](https://github.com/mosparo/mosparo/assets/114341346/46535f98-86b5-413e-b526-a89eeca34ec3)

![grafik](https://github.com/mosparo/mosparo/assets/114341346/e87620c7-6cb2-4170-bf75-450c3756e83d)

![grafik](https://github.com/mosparo/mosparo/assets/114341346/8d774fad-7d42-4456-b0e1-f2a39f246cdb)

![grafik](https://github.com/mosparo/mosparo/assets/114341346/0a37cce0-858a-4b49-a02b-25720744f70a)

What do I need to change to get this working again?

**Comment by zepich** [comment]
Hi @SuitDeer

Thank you very much for your issue.

I'm very sorry for the trouble. That's a tiny bug in the code that we overlooked because, normally, error messages should not be visible in the frontend in production.

We've just released v1.2.1, which solves this issue.

Additionally, we will verify with the next version that the Docker image does not show the error messages in the frontend so that such a situation can not happen again.

Kind regards,

zepich
